posix-configs/eagle/hil/px4.config

```diff
@@ -1,42 +1,16 @@
 uorb start
```

Comment:
These eagle/excelsior configs are probably not very useful in their entirety anymore.

Reply:
I want to get rid of posix-configs entirely soon.